cfg/Instructions.h

```diff
@@ -87,13 +87,14 @@ class Send final : public Instruction {
     VariableUseSite recv;
     core::NameRef fun;
     core::LocOffsets receiverLoc;
+    u2 numPosArgs;
```

Comment:
I found a send on pay-server with 3k positional arguments, which is why this is now a `u2` (it was creating a large frozen `Set`). It's not clear to me that we won't bump up against this again for other splat args functions, but perhaps 16k is reasonable?